string timestamp
int64 BE_num
float32[] SPEC_BE1
float32[] SPEC_BE2
float32[] SPEC_BE3
float32[] SPEC_BE4
float32[] SPEC_BE5
float32[] SPEC_BE6
float32[] SPEC_BE7
float32[] SPEC_BE8
float32[] SPEC_BE9
float32[] SPEC_BE10
float32[] SPEC_BE11
float32[] SPEC_BE12
float32[] SPEC_BE13
float32[] SPEC_BE14
float32[] SPEC_BE15
float32[] SPEC_BE16
float32[] SPEC_BE17
float32[] SPEC_BE18
float32[] SPEC_BE19
float32[] SPEC_BE20
string from_node
